### PR TITLE
changes how proxy is deployed on vercel

### PR DESCRIPTION
**Pull request by kasperp**
@mortenab this changes how the proxy is deployed on vercel it now uses vercel serverless functions https://vercel.com/docs/serverless-functions/introduction

### CODE REVIEW

File: ihear/src/api/fetch.ts [hunk 1]
```diff
@@ -31,13 +31,14 @@ export default (url: string, method: string, payload: object) => {
   headers.append('Accept', 'application/json');
   headers.append('Content-Type', 'application/json');
   headers.append('Access-Control-Allow-Origin', '*');
+  // headers.append('Authorization', 'ApiKey ' + process.env.REACT_APP_API_KEY);
```
Comment (mortenab) [inline ihear/src/api/fetch.ts:34]: 👀 should we keep this around?

Comment (kasperp) [reply ihear/src/api/fetch.ts:34]: yeah its useful for debugging locally

**Comment by mortenab** [comment]
I  am not sure how the API key is transferred by the serverless API - how does that work?

**Comment by kasperp** [comment]
They are stored on the vercel project and exposed as an environment variable 
image

**Review by mortenab — APPROVED** [review]

**Pull request merged**